.buildkite/pipeline.yml

```diff
@@ -50,3 +50,12 @@ steps:
       - nix-instantiate release.nix -A required-unstable-ghc8107-x86_64-darwin-native --show-trace --builders ''
     agents:
       system: x86_64-linux
+
+  - label: 'End-2-end test of hix project initialization and flakes development shell'
+    command:
+      - nix-shell -p cabal --run "cabal unpack hello"
+      - cd hello-*
+      - nix run "github:input-output-hk/haskell.nix#hix" -- init
+      - nix develop -c "cabal build"
```

Comment:
I think we might need a bash script, but something like this might allow us to run using the haskell.nix branch build kite is testing (rather than master):
```
HASKELL_NIX=$(pwd)
cd $(mktemp -d)
nix-shell -p cabal-install --run "cabal unpack hello"
cd hello-*
nix run $HASKELL_NIX#hix -- init
nix develop --override-input haskellNix $HASKELL_NIX -c "cabal build"
```

Reply:
Damn! You're right 😅